### ISSUE TITLE
Trial leaderboard gets occluded by the bottom bar

### ISSUE THREAD

**Issue by lazerwalker** [flappy-royale/flappy-royale#149]

**Comment by orta** [comment]
This was working for me today, perhaps there weren't enough birds to scroll?

**Comment by lazerwalker** [comment]
Yeah, seems like the real problem is the bottom is occluded by the back bar (when I filed that issue, the last player was half-covered by the bottom bar and I couldn't scroll to see all of them).

Suspect the fix here is, like, adding ~20px of padding-bottom or margin-bottom to the div.
